outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -106,50 +107,64 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
 
     }
 
-    private getCompanyLayout= () => {
-        const company = this.state.company as Company;
-
-        if (company.isEmpty())
+    private updateCompanyRequest = () => {
+        if (!this.props.partner.email)
         {
-            let enrichAndCreate = null;
-            if (this.props.partner.isAddedToDatabase() && !this.enriched)
+            let enrichmentInfo = new EnrichmentInfo(EnrichmentInfoType.EnrichContactWithNoEmail,
+                _t("This contact has no email address, no company could be enriched."));
+            this.context.showTopBarMessage(enrichmentInfo);
+            return;
+        }
+
+        this.setState({isLoading: true});
+
+
```

Comment:
Too many blank spaces.
I think the linter should tell you that?

Reply:
Yes, this has now been corrected.